database/migrations/platform/2023_10_09_000001_update_achievements_index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        // add the inverse indexes for user_id so lookups for untracked and deleted are more likely using indexes
+
+        Schema::table('Achievements', function (Blueprint $table) {
+            $sm = Schema::getConnection()->getDoctrineSchemaManager();
+            $indexesFound = $sm->listTableIndexes('Achievements');
+
+            if (!array_key_exists('achievements_game_id_published_index', $indexesFound)) {
+                $table->index(
+                    ['game_id', 'Flags'],
```

Comment:
This should be `GameID`.
```
SQLSTATE[42000]: Syntax error or access violation: 1072 Key column 'game_id' doesn't exist in table (SQL: alter table `Achievements` add index `achievements_game_id_published_index`(`game_id`, `Flags`))
```

Reply:
Thanks! Did those blind, should've double checked what I did in the database directly.
Fixed in https://github.com/RetroAchievements/RAWeb/pull/1910/commits/1e1117be4a9bcfa4d4893eca34eeb1785caaaef2